src/index.ts

```diff
@@ -191,12 +192,18 @@ export class FastAverageColor {
                 throw getError(`${options.algorithm} is unknown algorithm`);
         }
 
-        return algorithm(arr, len, {
+        const result = algorithm(arr, len, {
             defaultColor,
             ignoredColor: prepareIgnoredColor(options.ignoredColor),
             step,
             dominantDivider: options.dominantDivider,
         });
+        
+        if (options.alpha) {
```

Comment:
Sorry for the wait, this change overrides opacity. The same can be done by passing the result and the desired opacity to the public function `this.prepareResult()`.